GoogleTestAdapter/Core/Core.csproj

```diff
@@ -84,6 +84,8 @@
     <Compile Include="Model\TestProperty.cs" />
     <Compile Include="Runners\TestResultCollector.cs" />
     <Compile Include="Scheduling\SchedulingAnalyzer.cs" />
+    <Compile Include="Settings\ICTestPropertySettings.cs" />
```

Comment:
Same comment on this one. I think I'd call this something like ITestPropertySettings

Reply:
Yeah I think so. Why not just ITestPropertySettings?

Reply:
I can do that. Then the schema file changes too to just be called TestPropertySettings. There already is GoogleTestAdapterSettings.xsd (boost has similarly named settings). I guess TestPropertySettings is generic, but is it confusing? Like then why is there a TestPropertySettings in addition to a GoogleTestAdapterSettings?

Reply:
Can this same concept be put inside of GoogleTestAdapterSettings?

Reply:
Let me look into that. To augment what is already there.

Reply:
Actually, nevermind. I need to generate a separate section in the runsettings file (as I am now). But I'll rename that section to TestPropertySettings (getting rid of the "CTest" naming).

Reply:
Sounds good.